cloudbuild.yaml

```diff
@@ -11,6 +11,14 @@ steps:
       - PULL_BASE_REF=$_PULL_BASE_REF
     args:
       - release-staging
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20210722-085d930'
+    dir: 'hack/ccm'
+    entrypoint: make
+    env:
+      - PULL_BASE_REF=${_PULL_BASE_REF}
+      - HOME=/root
+    args:
+      - build-and-push-multi-arch
```

Comment:
but this will build both the images for every commit, we should create a separate cloudbuild.yaml for the ccm image alone and use that for the building image.

Reply:
to be more specific, create the file in hack/ccm folder itself and use that dir in the prow job like we are doing it for the e2e images in the [k/k/test/images](https://github.com/kubernetes/test-infra/blob/3ea0e0402f56805f427529b8d8a07c2c5cd68cb9/config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml#L40:L41)